WordPress/Classes/Services/TodayExtensionService.m

```diff
@@ -26,10 +26,10 @@ - (void)configureTodayWidgetWithSiteID:(NSNumber *)siteID blogName:(NSString *)b
     [sharedDefaults synchronize];
     
     NSError *error;
-    [SFHFKeychainUtils storeUsername:WPStatsTodayWidgetOAuth2TokenKeychainUsername
+    [SFHFKeychainUtils storeUsername:WPAppOAuth2TokenKeychainUsername
```

Comment:
I'm not sure how to test this, but it seems like the Today widget and the Share extension share the OAuth token, right?
I think this would break in the odd case where you can configure the Today widget for a Jetpack site connected to a second wp.com account.